motionplan/plannerOptions.go

```diff
@@ -175,6 +175,10 @@ type plannerOptions struct {
 	PlannerConstructor plannerConstructor
 
 	Fallback *plannerOptions
+
+	// relativeInputs is a flag that is set by the planning algorithm describing if the solutions it generates are
+	// relative as in each step in the solution builds off a previous one, as opposed to being asolute with respect to some reference frame.
+	relativeInputs bool
```

Comment:
How does this affect arms, if at all?

Reply:
Yes I think as written it is breaking arms because I never set it to false anywhere.  Good catch.  Hardware testing on those was going to be fun if you didnt find this